stories/List.js

```diff
@@ -32,8 +32,60 @@ storiesOf('List', module)
                     alert('Item was selected'); // eslint-disable-line no-alert, no-undef
                 }}
             >
-                <Checkbox id="item 4" />
+                <Checkbox id="item4" />
                 Clickable item
             </ListItem>
         </List>
-    ));
+    ))
+    .add('List showing job results', () => {
+        const onClickJob = e => {
+            e.preventDefault();
+            alert('Item was selected'); // eslint-disable-line no-alert, no-undef
+        };
+        const onClickAction = e => {
+            e.preventDefault();
+            e.stopPropagation();
+            alert('Action button was clicked'); // eslint-disable-line no-alert, no-undef
+        };
+
+        return (
+            <div style={{ width: '500px' }}>
+                <List isDivided>
+                    <ListItem onClick={onClickJob}>
+                        <JobResult
+                            title="My first job"
+                            location=" - Melbourne"
```

Comment:
@carlobernardini I was thinking this separator might be language specific. But if not, I will add it to the css as suggested above. Could you confirm?

Reply:
> I was thinking this separator might be language specific
Not that I know of,  but I'm not sure why CSS is the preferred way to do this. Depends on how opinionated you want that component to be. Then again you can also just deal with it in the component logic rather than CSS I guess (just prepend location with a dash if it's defined?)

Either way, it should be an *em dash* (—)  (make sure that it has white spacing on both sides so the dash is not 'attached' to the text)

Reply:
ok. added it via css